prow/cmd/tot/main_test.go

```diff
@@ -153,7 +153,7 @@ func TestFallback(t *testing.T) {
 var c *config.Config
 
 func TestMain(m *testing.M) {
-	conf, err := config.Load("../../config.yaml", "")
+	conf, err := config.Load("../../config.yaml", "../../../config")
```

Comment:
Should the second parameter be `../../../config/jobs`?
I think the BUILD file may need to be updated to make the `config` directory contents a dependency of this test.

Reply:
yeah we've bikesheded the subdir structure for a bit, will update this PR